runner/app/pipelines/audio_to_text.py

```diff
@@ -65,16 +65,23 @@ def __init__(self, model_id: str):
 
         torch_device = get_torch_device()
 
-        # Get the GPU device properties
-        device = torch.cuda.get_device_properties(0)
-        major, minor = device.major, device.minor
-
-        # Compute capability of Ampere starts at 8.0
-        if (major, minor) >= (8, 0):
-            attn_implementation = "flash_attention_2"
+        # Enable FlashAttention based on device compatibility.
+        attn_implementation = "eager"
+        if torch_device.type == "cuda":
```

Comment:
@RUFFY-369 I added some code to ensure that CPU inference still works.

Reply:
Oh! Thanks for adding it. :+1: